pontoon/base/models.py

```diff
@@ -255,6 +368,23 @@ def add_translation(self, string, locale, user):
         translation.save()
 
 
+class TranslationManager(models.Manager):
+    def get_queryset(self):
+        return (super(TranslationManager, self).get_queryset()
+                .filter(deleted__isnull=True))
+
+
+class DeletedTranslationManager(models.Manager):
+    def get_queryset(self):
+        return (super(DeletedTranslationManager, self).get_queryset()
+                .filter(deleted__isnull=False))
+
+
+def extra_default():
+    """Default value for the Translation.extra field."""
+    return {}
```

Comment:
See https://docs.djangoproject.com/en/1.8/ref/models/fields/#default. Lambdas can't be serialized for migrations, nor can methods that aren't defined in the top level of the module. Thus, I use a top-level function that can be serialized.